docs/SUMMARY.md

```
@@ -1,20 +1,57 @@
+* [Overview](getting-started/overview.md)
+* [Installation](getting-started/installation.md)
+* [Your First Mock Server](getting-started/your-first-mock.md)
+* [Dynamic Responses](getting-started/dynamic-responses.md)
+* [Multiple Endpoints](getting-started/multiple-endpoints.md)
+* [Real-World Example](getting-started/real-world-example.md)
+
+## Core Concepts
+
+Learn the key concepts and understand how moclojer works.
+
+### Templates
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)